vic/drivers/python/setup.py

```diff
@@ -86,6 +88,94 @@ def write_version_py(filename=None):
 
 
 # -------------------------------------------------------------------- #
+def maybe_eval_string(s):
+    '''run eval on strings between brackets e.g. [20 + 3] becomes [23]'''
+    matches = re.findall("\[(.*?)\]", s)
+    for match in matches:
+        if match:
```

Comment:
It saves going through a try except statement when `matches = []`.  So no we don't need it but I'm going to keep it.